src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +112,7 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={!allRequiredFieldsAreFilled && true}
```

Comment:
Why did you add boolean here?

```suggestion
            disabled={!allRequiredFieldsAreFilled}
```

Reply:
well...then I don't know)))